.gitignore

```diff
@@ -163,4 +163,5 @@ dist
 .sourcebot
 /bin
 /config.json
-.DS_Store
\ No newline at end of file
+.DS_Store
+.*.sw*
```

Comment:
What are `.sw` files?

Reply:
I generally only use an IDE for Java.  These are state files that 'vim' creates when a file is being edited.  I don't like to see them in 'git status' output, and certainly would not want to check them in by accident.

Vim usage is why I added the .editorconfig file as well.  I didn't want tabs creeping into your code base.